### ISSUE TITLE
Inconsistent translation of "Querschnittskonzepte"

### ISSUE THREAD

**Issue by ulibecker** [isaqb-org/curriculum-foundation#168]
In chapter 1, the german word "Querschnittskonzepte" is translated differently:

Terms: "cross-cutting concerns"
LG 1-1: "cross-cutting concepts"

We should use "cross-cutting concepts" in both places

**Comment by ulibecker** [comment]
This also affects LG 3-6:
"Querschnittskonzepte" in German, "cross-cutting concerns" in English

**Comment by gernotstarke** [comment]
replaced "cross-cutting concerns" by "cross-cutting concepts" in 

LG 1-6, LG 2-4, LG 3-6

**Comment by rhoadesre** [comment]
I see that Gernot changed "cross-cutting concerns" to "cross-cutting concepts".

The term that I've always used and read in references was "cross-cutting concerns", not "... concepts". I first heard the term "cross-cutting concepts" when I came to iSAQB.

Therefore, I would like to reopen this issue and change the term to "cross-cutting concepts". 

Any strong objections?

**Comment by gernotstarke** [comment]
yes - as arc42.org uses "cross-cutting concepts".

As it's quite heavily used in practice, I consider it to be a valid source (disclaimer: I'm co-author...)

**Comment by programming-wolf** [comment]
Any progress on this?

**Comment by gernotstarke** [comment]
no, so let's use cross-cutting concepts. Nobody has opposed, consider it decided then.

**Comment by alxlo** [comment]
IMHO cross cutting concepts is what you can use to adress cross cutting concerns. I see a difference there. 

Am 4. Februar 2022 15:53:44 MEZ schrieb "Dr. Gernot Starke" ***@***.***>:
>no, so let's use cross-cutting concepts. Nobody has opposed, consider it decided then.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/isaqb-org/curriculum-foundation/issues/168#issuecomment-1030060470
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by alxlo** [comment]
Did a read-up of the curriculum and in the current context of usage cross-cutting concepts and Querschnittskonzepte is fine by me. 

_However, we should be sure to have the same understanding of the distinction between cross-cutting concepts and cross-cutting concerns here (at the very least for future edits)._

The Trainer's Guide includes the following background information for a slide concerning LG2-4  (... after fixing a lot of confusing typos that are still in the latest 0.9.5 version of the Trainer's Guide):
- "Students should be able to explain what cross-cutting concerns are and how cross-cutting concepts provide solution strategies for such concerns. "
- "You will need cross-cutting concepts to deal with cross cutting concerns."
- "There might be discussions about whether to flavour this segment as concepts or concerns. From a practical point of view this discussion is futile. There are cross-cutting concerns in almost any architecture design and you need cross-cutting concepts to address them. "

So I'm in favour of @gernotstarke 's proposal to leave it at "concepts" in the curriculum and hope everyone shares the understanding of the terms the way I convey them in the Trainer's Guide.

**Comment by rhoadesre** [comment]
Hello everyone,

ok. My two cents worth. I still would like to see “cross-cutting concerns” rather than “cross-cutting concepts”, but in the end it’s not that important to me and can live with “cross-cutting concepts”.

For me, “cross-cutting concerns” are the same as / synonymous with “cross-cutting concepts”. Therefore, use either cross-cutting concerns or cross-cutting concepts, definitely not both. I would be very against using these two terms to mean different things. The three examples below will only serve to confuse everyone and everything.

The way that “cross-cutting concerns” are used in the examples below implies stakeholder concerns, which is an entirely different topic. A cross-cutting concept is a set of functionality that cannot be implemented in one building block, but per definition impacts multiple building blocks at the implementation level. What would be a “cross-cutting concern” as described below? Stakeholder concerns that cross multiple business functions? We also don’t design a solution for stakeholder concerns unless they are specifically specified as requirements. Please, please, please don’t mix these two terms. It will only lead to confusion.

A possible formulation that I could accept for the statements below is:

  *   The implementation of certain requirements may impact multiple building blocks (components, classes, libraries, etc.).  So called “Cross-Cutting Concepts” provide an implementation solution to such requirements. Students should be able to explain which cross-cutting concepts typically are required in systems (e.g., integration, exception handling, logging, security, etc.) as well as different approaches for their implementation.
  *   Students should take into account that both the implementation as well as the documentation of the cross-cutting concepts should be changed locally, if possible. For example, to change the logging implementation, it is only necessary to make changes to code in one location and not in every component and class that requires logging.

Mit freundlichen Grüßen,

Roger Rhoades

***@***.***

+49 152 53 44 33 55
***@***.******@***.***>
www.albion.eu<http://www.albion.eu/>

**Comment by ulibecker** [comment]
> Did a read-up of the curriculum and in the current context of usage cross-cutting concepts and Querschnittskonzepte is fine by me.
> 
> _However, we should be sure to have the same understanding of the distinction between cross-cutting concepts and cross-cutting concerns here (at the very least for future edits)._
> ...
> * "You will need cross-cutting concepts to deal with cross cutting concerns."
> ...

I fully agree with the differentiation between "concerns" and "concepts" that @alxlo mentioned from the trainer guide. It's fine for me to leave that in the curriculum as it currently is.

**Comment by gernotstarke** [comment]
is now consistently translated, therefore  closed
